docs/Manual/10 Packages.md

```diff
@@ -2,45 +2,45 @@
 [//]: # (Version: v1.2.1)
 
 # Section 10: Building Packages
-The Accera `Package` class represents a collection of Accera-generated functions. When a package is built, it creates a stand-alone function library that can be used by other pieces of software. Accera currently supports two package formats: HAT and MLIR.
+The `Package` class in Accera represents a collection of Accera-generated functions. Whenever a package is built, it creates a stand-alone function library that other pieces of software can use. Currently, Accera supports two package formats: HAT and MLIR.
 
 ## HAT package format
-[HAT](https://github.com/microsoft/hat) is a format for packaging compiled libraries in the C programming language. HAT stands for "Header Annotated with TOML", which implies that a standard C header is decorated with useful metadata in the TOML markup language.
+[HAT](https://github.com/microsoft/hat) "Header Annotated with TOML" is a format for packaging compiled libraries in the C  programming language. HAT implies that a standard C header is styled with useful metadata in the TOML markup language.
 
-Say that `nest` contains some loop-nest logic. To build a HAT package that contains a function with this logic for the Windows operating system, we do the following:
+Consider that `nest` holds a loop-nest logic. To build a HAT package containing a function with this logic for Windows Operation System, we need to write the following lines of code: 
 ```python
 package = acc.Package()
 package.add(nest, base_name="func1")
 package.build(format=acc.Package.Format.HAT_DYNAMIC, name="myPackage", platform=acc.Package.Platform.WINDOWS)
 ```
 
-The result is two files: `MyPackage.hat` and `MyPackage.lib`. The output directory defaults to the current working directory. To change the output directory:
+The result has two files: `MyPackage.hat` and `MyPackage.lib`. The output directory defaults to current working directory. Though we can change the output directory with this:
```

Comment:
"We can change the output directory with `output_dir` set to a relative or absolute path:"